### ISSUE TITLE
Metatag module is missing

### ISSUE THREAD

**Issue by thejimbirch** [MidCamp/midcamp#99]
Add and configure metatag module.

**Comment by kthull** [comment]
@thejimbirch by configure, do you mean add the metatag field to various content types?

**Comment by thejimbirch** [comment]
Yes. If you need a jump start, or guidance, I created a module that loads a
bunch of preconfigured config files:

https://www.drupal.org/project/seo_starter
